naveenG/task2/sample.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>sample</title>
+    
+</head>
+<body>
+
+ 
+ <table id="table" border="">
+    <tr>
+        <th>S.NO</th>
+        <th><span id="Mem">Member of Institude</span></th>
+        <th><span id="name1">Candidate</span></th>
+        <th><span id="name2">Fathername</span></th>
+        <th><span id="name3">Mothername</span></th>
+        <th><span id="year">completeyear</span></th>
+        <th><span id="adde">Address</span></th>
+        <th><span id="email">Mail Id</span></th>
+        <th><span id="mobile1">Mobile number</span></th>
+        <th><span id="mobile">Mobile</span></th>
+        <th><span id="status">Status</span></th>
+        <th><span id="work">Work</span></th>
+        <th><span id="card">Id Proof</span></th>
+        
+    </tr>
+ 
+<script>
+    var ls = localStorage.getItem("value1");
```

Comment:
Move this Script code to separate JS file and provide link in the HTML page.
Indent the code properly in JS file.

Reply:
I gives the separate JS file for sample page and linked to the HTML page and also I given the indent for JS code.